clion-plugin/src/test/kotlin/org/utbot/cpp/clion/plugin/BaseGenerationTestCase.kt

```diff
@@ -98,7 +98,7 @@ abstract class BaseGenerationTestCase {
     /**
      * Waits until all requests initiated during tests are finished
      */
-    fun waitForRequestsToFinish() = runBlocking {
+    fun waitForRequestsToFinish() {
```

Comment:
Why did we remove runBlocking here?

Reply:
it is not needed here: there are no suspending calls inside function body, the runBlocking is inside `Client`'s `waitForRequestsToFinish`. This is one is most likely an old code